### PR TITLE
During application, creation create a demo template

### LINKED ISSUE #284: [RLD-121] - During application, creation create a demo template
## Why?

We will present a default template so the user won't need to build his first template on the onboarding.

Notification Content

Channels: In-App

CTA: Link to the platform with the user token in the query.

## How?

By saving the default template on the application creation.

### Other

Mark template as "demo"

### PR DESCRIPTION
**Pull request by djabarovgeorge**
Closes #284

### CODE REVIEW

File: packages/cli/src/commands/init.ts [hunk 1]
```diff
@@ -71,6 +81,41 @@ async function createApplicationHandler(config: ConfigService, answers: Answers)
...
+        data: {
+          url: redirectUrl,
+        },
+      },
+    },
+  ];
+
+  return {
+    notificationGroupId,
+    name: 'demo notification name',
```
Comment (scopsy) [inline packages/cli/src/commands/init.ts:110]:
```suggestion
    name: 'On-boarding notification',
```

File: packages/cli/src/commands/init.ts [hunk 2]
```diff
@@ -71,6 +81,41 @@ async function createApplicationHandler(config: ConfigService, answers: Answers)
...
+  await createNotificationTemplates(template);
+}
+
+function buildTemplate(notificationGroupId: string): ICreateNotificationTemplateDto {
+  const redirectUrl = `${CLIENT_LOGIN_URL}?token={{token}}`;
+
+  const messages = [
+    {
+      type: ChannelTypeEnum.IN_APP,
+      content: 'Test content for <b>{{firstName}}</b>',
```
Comment (scopsy) [inline packages/cli/src/commands/init.ts:98]:
```suggestion
      content: 'Welcome <b>{{firstName}}</b>! This your first notification, click on it to visit your live dashboard',
```

**Review by scopsy — APPROVED** [review]
Looks great and very clean! Added some content changes

File: packages/cli/src/commands/init.ts (hunk 2 above)
Comment (djabarovgeorge) [reply packages/cli/src/commands/init.ts:98]: Nice catch! I wanted to add 'click me' context as well so it will be more intuitive to the user!

**Pull request merged**